model/src/main/proto/spine_examples/shareaware/wallet/wallet.proto

```diff
@@ -54,13 +55,16 @@ message Wallet {
     //
     spine.money.Money balance = 2;
 
-    // An amount of reserved money.
+    // The binding of `WithdrawalId` to the amount of money reserved by it.
```

Comment:
"Binding" is too technical. How about this:

"The reserved amounts of money per operation which performed the reservation."